### ISSUE TITLE
(Unrelated) Do you have a Monero mining rig?

### ISSUE THREAD

**Issue by PPPDUD** [benthetechguy/nix#1]
I spotted someone with the username "BenTheTechGuy" getting a small Monero reward on my share on the p2pool Mini chain (**https://mini.p2pool.observer/share/16e0b655e870bfbaa4c1a24a293e471cea512349808e532d8b24ab2a5a6c1297**). Was that you, and do you have a mining rig, or is it someone else?

If you do have a mining rig, what cryptomining software are you running? I'm using the Monero GUI.

**Comment by benthetechguy** [comment]
Yes, that was me! When I'm not using them, I leave my M1 Mac Mini and [workstation/gaming PC](https://pcpartpicker.com/list/xJs2BL) on and mine on P2Pool Mini with xmrig. The Mac Mini runs monerod in the background with the blockchain being stored on a thunderbolt 1TB SSD. The workstation is running NixOS and the Mac Mini runs Fedora Asahi Remix, but I reboot into macOS sometimes.

**Comment by PPPDUD** [comment]
Cool! I use it on a Windows 11 Pro custom-built desktop and store the blockchain on a T7 Shield (2TB).
